### DISCUSSION TITLE
about my pull request#7401

### DISCUSSION THREAD

**Discussion by VersaillesHeroes** [consenlabs/webview#63] [General]
Dear Imtoken support,

My pull request#7401 is for Imtoken branch: [consenlabs/token-profile](https://github.com/consenlabs/token-profile).

Can this branch admin help with me to merge it to main branch?

Regards,

Versailles Heroes Team

**Comment by xwartz** [comment]
You should create the PR at [consenlabs/token-profile](https://github.com/consenlabs/token-profile), not here.

**Reply by VersaillesHeroes** [reply]
I already created new PR#7401 under it but no one help to merge it to main branch
